src/components/staking/LiValidator.vue

```diff
@@ -54,13 +57,17 @@
       }}
     </td>
     <td :class="{ 'hide-xs': showOnMobile !== 'voting-power' }">
-      {{ validator.votingPower | percent }}
+      <span v-if="validator.votingPower">
+        {{ validator.votingPower | percent }}
+      </span>
+      <span v-else>{{ validator.votingPower | noBlanks }}</span>
```

Comment:
this doesn't really make sense.

if `!validator.votingPower` why would we display `validator.votingPower` with a filter?

validators should _always_ have voting power.  even if it's 0 (which it is for cosmos validators who are not in the set, for example) then the percent filter should just display/return 0. 

i don't think we need extra HTML or conditional logic here at all.

Reply:
0 means the validators has 0 voting power and '--' means "this feature is not available in this network. I personally think '--' is a better option, more explanatory.
Also, this is an outdated change. Now this is implemented much more beautifully with the chooseFilter method in just a single line of code.

Reply:
livepeer doesn't have something like a stake distribution??! sounds wrong.

Reply:
please push the latest changes so we can discuss only relevant things.

Reply:
There is no voting and there are no proposals in Livepeer

Reply:
`votingPower` has limited to do with voting on proposals. this is the percent of all stake delegated to this validator

Reply:
ok, will change it. I didn't know that is what it actually meant